src/utils/hookts.ts

```diff
@@ -60,7 +60,7 @@ export const useClientEffect = (
   dependencies?: DependencyList
 ): void => {
   if (typeof window !== "undefined") {
-    useEffect(effect, dependencies)
+    useEffect(effect, dependencies) // eslint-disable-line
```

Comment:
@GemN @louisholley this actually violates the hook rule that hooks can't be conditional. wdyt?

Reply:
i think it's fine in this case, `typeof window` will never go from undefined to undefined or vice versa so shouldn't give us any issues